src/org/exist/jetty/ServerShutdown.java

```diff
@@ -120,35 +112,30 @@ public static void main(String[] args) {
     }
 
     private static Properties loadProperties() {
-
-        final Properties clientProps = new Properties();
-        final Path propFile = ConfigurationHelper.lookup("client.properties");
-        InputStream pin = null;
-
-        // Try to load from file
+        final Properties properties = new Properties();
         try {
-            pin = Files.newInputStream(propFile);
-        } catch (final IOException ex) {
-            // File not found, no exception handling
-        }
-
-        if (pin == null) {
-            // Try to load via classloader
-            pin = ServerShutdown.class.getResourceAsStream("client.properties");
-        }
-
-        if (pin != null) {
-            // Try to load properties from stream
+            InputStream pin = null;
             try {
-            	try {
-            		clientProps.load(pin);
-            	} finally {
-            		pin.close();
-            	}
-            } catch (final IOException ex) {
-                //
+                final Path propFile = ConfigurationHelper.lookup("client.properties");
+
+                if (Files.isReadable(propFile)) {
+                    pin = Files.newInputStream(propFile);
+                } else {
+                    pin = ServerShutdown.class.getResourceAsStream("client.properties");
+                }
+
+                if (pin != null) {
+                    properties.load(pin);
+                }
+
+            } finally {
+                if (pin != null) {
+                    pin.close();
+                }
             }
+        } catch (final IOException e) {
+            e.printStackTrace();
```

Comment:
No, I wanted the fact the `client.properties` file is missing to be a non-fatal error.